test-source/test/ceylon/time/testDates.ceylon

```diff
@@ -141,7 +141,7 @@ shared test void testDateMinusMonths() {
 }
 
 shared test void testDateMinusLessDays() {
-    assertEquals( date(1982,november,30), date(1982,december,31).minusMonths(1));
+    assertEquals( date(1982,december,31).minusMonths(1), date(1982,november,30) );
```

Comment:
These could also be converted to named argument invocation style.
I find I've come to like it much more than ordinal invocation. Specially for test assertions.

this particualtr one would look much better  imho if it was formatted like this:
```ceylon
shared test void testDateMinusLessDays() => assertEquals {
    actual = date(1982,december,31).minusMonths(1);
    expected = date(1982,november,30);
};
```